lib/merge.ts

```diff
@@ -402,6 +421,37 @@ export function normalizeSourceFile(f: ts.SourceFile, fc: FacadeConverter) {
     }
   }
 
+  function findPropertyInClass(propName: string, classLike: base.ClassLike): ts.ClassElement|
+      undefined {
+    const members = classLike.members as ts.NodeArray<ts.ClassElement>;
+    return members.find((member: ts.ClassElement) => {
+      if (base.ident(member.name) === propName) {
+        return true;
+      }
+    });
+  }
+
+  function findPropertyInHierarchy(
```

Comment:
sure there aren't existing ts methods that can perform these lookups?

Reply:
I wasn't able to find anything